src/pages/Loans/LoansOverview/utils/get-max-borrowable-amount.tsx

```diff
@@ -22,15 +24,23 @@ const getMaxBorrowableAmount = (
     return newMonetaryAmount(0, asset.currency);
   }
 
-  const { availableCapacity, currency } = asset;
+  const { availableCapacity, currency, borrowCap, totalBorrows } = asset;
 
   const availableCollateralUSDValue = totalCollateralAmountUSD.sub(totalBorrowedAmountUSD);
   const maxBorrowableCurrencyAmount = availableCollateralUSDValue.div(assetPrice);
 
   const maxBorrowableAmountByCollateral =
     new MonetaryAmount(currency, maxBorrowableCurrencyAmount) || new MonetaryAmount(currency, 0);
 
-  return availableCapacity.gt(maxBorrowableAmountByCollateral) ? maxBorrowableAmountByCollateral : availableCapacity;
+  const maxBorrowableAmount = pickSmallerAmount(
+    pickSmallerAmount(availableCapacity, borrowCap.sub(totalBorrows)),
```

Comment:
Could you extract this to a variable? Its hard to read it like this.

Reply:
done, is it easier to read now?